gpu-operator/platform-support.rst

```diff
@@ -461,7 +461,7 @@ Support for GPUDirect RDMA
 
 Supported operating systems and NVIDIA GPU Drivers with GPUDirect RDMA.
 
-- Ubuntu 20.04 and 22.04 LTS with Network Operator 23.10.0
+- Ubuntu 20.04 and 22.04 LTS with Network Operator 24.1.0
```

Comment:
A few open questions:
- are we bumping the Network Operator version we support on OpenShift as well?
- are we bumping the Network Operator version we support with GDS as well?